docs/change_controlled.rst

```diff
@@ -0,0 +1,78 @@
+###################################
+Writing Change-Controlled Documents
+###################################
+
+Change-controlled documents are documents that require approval from the Change Control Board (CCB) before they can be released.
+The project-wide change control process is described in `LPM-19`_, with the process for managing documents being described in `LPM-51`_, and covers those documents using DocuShare handles "LSE" or "LPM", that must be approved by the Project `CCB`_.
+For Data Management, these documents have the DocuShare handle "LDM" and they must be approved by the DM CCB using the :ref:`RFC <decision-making-rfc>` process.
+These change-controlled documents can be written using the Word templates defined in `Document-9224`_, but the preference within Data Management is to write change-controlled documents in LaTeX using the ``lsstdoc`` document class provided by the `lsst-texmf`_ package, and to develop these documents using ``git`` repositories within the LSST organization on `GitHub`_.
+
+.. _CCB: https://project.lsst.org/groups/ccb/
+.. _Document-9224: https://ls.st/Document-9224
+.. _lsst-texmf: https://lsst-texmf.lsst.io
+.. _LPM-19: https://ls.st/LPM-19
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. note::
+  The definition of which types of documents should be change-controlled is defined in `LDM-294`_.
+
+
+Developer Workflow
+==================
+
+The development of change-controlled documents differs from :ref:`that of code <git-branching>`, in that some documents under development can take a long time to evolve and can be developed under multiple stories (ticket branches), and by multiple authors.
+Repositories associated with change-controlled documents should therefore use two protected branches: ``master`` should correspond solely to an archived instance of the document on DocuShare, and ``draft`` is a development branch that should be used as the document evolves.
+Ticket branches should branch from ``draft`` and should follow standard :ref:`DM branching policies <git-branching>`.
```

Comment:
To deal with the multiple-RFC problem in the "warning" below, I suggest that specific RFC-triggered updates go into `tickets/DM-NNNN` branches from `master`; only long-lived, multi-author, pre-RFC work would go into `draft`.  Hopefully there aren't two instances of the latter at the same time.